src/pastelid/ed.h

```diff
@@ -415,4 +415,25 @@ namespace ed_crypto {
     using key_dh448 = key<EVP_PKEY_X448>;
     using key_dsa25519 = key<EVP_PKEY_ED25519>;
     using key_hd25519 = key<EVP_PKEY_X25519>;
+
+    static bool isValidPassphrase(std::string pastelid, SecureString strKeyPass)
+        {
+            try {
+                //Get pastelkeyfile
+                fs::path pathPastelKeys(GetArg("-pastelkeysdir", "pastelkeys"));
+                pathPastelKeys = GetDataDir() / pathPastelKeys;
+
+                if (!fs::exists(pathPastelKeys) ||
+                    !fs::is_directory(pathPastelKeys)) {
+                    fs::create_directories(pathPastelKeys);
```

Comment:
return 'false' if directory is not exist?

Reply:
I just realized there is no need to check directory.

Reply:
agree